tests/model_fields/tests.py

```diff
@@ -661,6 +689,15 @@ def test_backend_range_validation(self):
             }
             with self.assertRaisesMessage(ValidationError, expected_message):
                 instance.full_clean()
+            try:
+                with transaction.atomic():
+                    instance.save()
+            except (DataError, IntegrityError):
+                pass  # backends such as Postgres don't fail silently
+            else:
+                qs = self.model.objects.filter(value__lte=min_value)
```

Comment:
SQLite also mimics the behavior, not sure about Oracle. The intention is to test that the ranges that are set for validators are inline with the DB. There may be exceptions to this with future DBs, but the assumption is that it exhibits one of the two behaviors.